site/src/util/workElections.ts

```diff
@@ -27,6 +27,7 @@ export const findJurisdiction = ( state: string, county?: string, city?: string
          `${city} village, ${county} County`,
          `${city} city, ${county} County`,
          `${city} (City)`,
+         `${city} City`,
```

Comment:
This is to support New York City which is confusingly not called New York (City) like every other city